README.md

```diff
@@ -3,19 +3,38 @@ Goal: To bootstrap a light-weight Jenkins instance with auto-scaling replicas us
 
 *NOTE: This is a work in progress! The README will be updated with functional steps as they are added.*
 
-## Create AWS infrastructure
-Running this setup script will create a security group in your chosen VPC, an EC2 instance,
-and a DynamoDB table and s3 bucket for remote terraform state management.
+## Replace custom variables for your AWS account
+Update the following values in `terraform/backend.auto.tfvars`: 
```

Comment:
Did you mean to delete this file?   Or change the documentation to reflect creating it from scratch?

Reply:
It appears there is another one in `terraform/remote_state/`

Reply:
I had the wrong path